readme.md

```diff
@@ -18,15 +18,15 @@ To learn about getting started with other Origami components see the [Origami co
 ## Versions
 
 |version|valid js|valid sass|valid html|valid demos|valid readme|js tests pass|sass tests pass|
-|-------|--------|----------|----------|-----------|------------|-------------|---------------|
+|***---|******|******--|******--|******---|*********|*********-|*********---|
```

Comment:
This looks like a find and replace mistake as the table no longer has column headings